Makefile

```diff
@@ -26,6 +26,14 @@ install_ci: install
 		go get golang.org/x/tools/cmd/cover
 
 
+.PHONY: update_man
+update_man:
+	go install .
+	yab --man-page > man/yab.1
```

Comment:
This PR adds the `man` dir with the generated man pages so anyone running this command after this change will already have the directory